src/app/api/repositories/[owner]/[repo]/preferences/route.ts

```
@@ -0,0 +1,63 @@
+    if (Array.isArray(ignoredTypes)) {
+      finalIgnoredTypes = JSON.stringify(ignoredTypes);
+    } else if (typeof ignoredTypes === "string") {
+      finalIgnoredTypes = ignoredTypes;
+    }
+
+    const dbUser = await prisma.user.findUnique({
+      where: { email: session.user.email },
+    });
+    if (!dbUser) return new Response("User not found", { status: 404 });
+
+    const updateData: Record<string, any> = {};
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)